event_database/cloud_functions/process-transfer.go

```diff
@@ -217,11 +217,6 @@ func ProcessTransfer(ctx context.Context, m PubSubMessage) error {
 			nativeTokenAddress = string(item.Value)
 		}
 	}
-	if coinId == "" {
```

Comment:
just wondering, do we want to write transfer details to the row if we do not have coinId or price?  I'm wondering if that will have downstream effects, code that expects a coinId or price if the transfer details family exists.

Reply:
The downstream/calculate functions now look up the CoinGecko API ID in the allowlist and don't depend on the one set in the transfer details column family or the price. I think that it should be okay for the transfer details column family to exist even if the coinId or price is empty/0